src/main/java/org/example/game/Game.java

```diff
@@ -1,85 +1,104 @@
 package org.example.game;
 
-import org.example.dto.GameStepDTO;
+import org.example.dto.GameStateDTO;
+import org.example.dto.StepForMoveDTO;
+import org.example.myExeptions.MyCustomExceptions;
 
 import java.util.Arrays;
 import java.util.Objects;
 
 public class Game {
-    private String[][] gameField;
-    private Sign sign = Sign.X;
-    private int fieldSize = 3;
+    private CellValue cellValue = CellValue.X;
```

Comment:
Here you are mixing two concepts, cell value and sign. CellValue can be X, Y or Null. A sign can be X or Y. You can create a converter for convenience.
After that, you create the game field, but instead of using a two-dimension array of CellValues, you are using Strings. It leads to potential mistakes and forces you to handle unwanted edge cases.